open_parser/base.py

```diff
@@ -0,0 +1,69 @@
+import json
+
+import requests
+
+CAMBIO_UPLOAD_URL = (
+    "https://p1iz3c1c77.execute-api.us-west-2.amazonaws.com/v1/cambio_api/upload"
+)
+CAMBIO_EXTRACT_URL = (
+    "https://p1iz3c1c77.execute-api.us-west-2.amazonaws.com/v1/cambio_api/extract"
+)
+
+
+class OpenParser:
+    def __init__(self, apiKey="") -> None:
+        self._uploadurl = CAMBIO_UPLOAD_URL
+        self._extracturl = CAMBIO_EXTRACT_URL
+        self._request_header = {"x-api-key": apiKey}
+
+    def setAPIKey(self, apiKey):
+        self._request_header = {"x-api-key": apiKey}
+
+    def extract_pdf_content(self, file_path):
```

Comment:
nit: I would suggest to rename to `extract` because our backend can support more than pdf.